### ISSUE TITLE
Feature request: Warn people that this is insecure on first run

### ISSUE THREAD

**Issue by Juul** [TryQuiet/quiet#2588]
I would put a big scary warning that pops up on first use to advise people that they should consider the app insecure (for now).

**Comment by holmesworcester** [comment]
I agree with this. We had a recent indication that a user missed the warning on the website. We'll add one. 

"Quiet is in beta and shouldn't be used for activities requiring security."

**Comment by jgaylor** [comment]
@holmesworcester [here are some options](https://www.figma.com/design/RuFXslbqmGwi7nNgB90r4T/Template-(Copy-this)-(Copy)?node-id=2811-7539&t=mNfqHjyGKNCoOxa3-1), unless you think it should be on the loading screen. I was imagining this as a permanent message on the join view, but it could also be a snackbar like alert that needs to be dismissed i suppose. Anyway, just a quick pass.

**Comment by holmesworcester** [comment]
@jgaylor I like all of these, option A is my first choice, and yet none of them really reach the bar of "*big, scary* warning" so let's make the text bigger and hard to miss.

**Comment by jgaylor** [comment]
Works for me. Here's [option A2 with text that matches the size and weight of the main actions](https://www.figma.com/design/RuFXslbqmGwi7nNgB90r4T/Beta-security-warning?node-id=2811-7539&t=WcLEKeMSEcHxkRL1-1).

**Comment by holmesworcester** [comment]
This option is great.

**Comment by holmesworcester** [comment]
If we add this now, we'd use the new getting started screen with only two options: join community and create community. Let's make a design for that just so it's clear.

To be specific:

When the user opens the app without being part of a community she will see the new getting started screen with two options: join community & create community.

The warning would also display here.

These options will take her to the *existing* screens for joining and creating communities, and everything else will be left as is.

We would implement this on desktop and mobile.

**Comment by holmesworcester** [comment]
https://www.figma.com/design/RuFXslbqmGwi7nNgB90r4T/Beta-security-warning?node-id=2841-1782&t=Q9eLKC1NoTNNTivr-4

**Comment by holmesworcester** [comment]
@jgaylor let's add this to all of our prototypes.

**Comment by jgaylor** [comment]
Ok, moving to design in progress to queue it up to add. Also, not sure what "All prototypes" means, we should talk specifics and specific links to the ones where it's needed.

**Comment by holmesworcester** [comment]
It's the "get started" prototype: https://www.figma.com/proto/f6Nr5b5wtvk6Xoh1HJZ8Dd/Get-started-(prototype)?node-id=2811-2550&p=f&t=Qw2661SfoZlCfXgD-0&scaling=min-zoom&content-scaling=fixed&page-id=7%3A2510&starting-point-node-id=2811%3A2550

**Comment by jgaylor** [comment]
@holmesworcester thank you. [Get started prototype](https://www.figma.com/proto/f6Nr5b5wtvk6Xoh1HJZ8Dd/Get-started-(prototype)?page-id=7%3A2510&node-id=2811-2550&viewport=57%2C-173%2C0.25&t=qz1m7jJ03jc9y6D3-1&scaling=min-zoom&content-scaling=fixed&starting-point-node-id=2811%3A2550&show-proto-sidebar=1) has been updated.

It occurred to me that we might want this beta warning on the **Join screen** as well—in the [Join from invite link prototype](https://www.figma.com/design/dSEZJr9crJjcV3ILogea9C/Join-from-invite-link-(prototype)?node-id=7-2510&t=7DLX658N5NdvyLzi-1). In THIS user flow I believe we bypass the "get started" screen, and user wouldn't see the warning otherwise. Wdyt?

**Comment by holmesworcester** [comment]
> It occurred to me that we might want this beta warning on the Join screen as well—in the [Join from invite link prototype](https://www.figma.com/design/dSEZJr9crJjcV3ILogea9C/Join-from-invite-link-(prototype)?node-id=7-2510&t=7DLX658N5NdvyLzi-1). In THIS user flow I believe we bypass the "get started" screen, and user wouldn't see the warning otherwise. Wdyt?

Yes, we should add it to that screen too.

**Comment by jgaylor** [comment]
@holmesworcester This is now in the master library as a component and appears on both prototypes:
1. [Get started prototype](https://www.figma.com/proto/f6Nr5b5wtvk6Xoh1HJZ8Dd/Get-started-(prototype)?page-id=7%3A2510&node-id=2811-2550&viewport=57%2C-173%2C0.25&t=qz1m7jJ03jc9y6D3-1&scaling=min-zoom&content-scaling=fixed&starting-point-node-id=2811%3A2550&show-proto-sidebar=1)
2. [Join from invite link prototype](https://www.figma.com/design/dSEZJr9crJjcV3ILogea9C/Join-from-invite-link-(prototype)?node-id=7-2510&t=7DLX658N5NdvyLzi-1)

If nothing else, it can go to dev backlog, or wherever you see fit.

**Comment by Ccocconut** [comment]
> Works for me. Here's [option A2 with text that matches the size and weight of the main actions](https://www.figma.com/design/RuFXslbqmGwi7nNgB90r4T/Beta-security-warning?node-id=2811-7539&t=WcLEKeMSEcHxkRL1-1).

I prefer the last two, but I think the warning sign should be colored like it is in the first one. I do not mind the first one either.

**Comment by kingalg** [comment]
Version: 7.4.1-alpha.1
Done
